codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/parse/EventStreamUnmarshallerGenerator.kt

```diff
@@ -54,7 +54,7 @@ class EventStreamUnmarshallerGenerator(
         "Error" to RuntimeType("Error", smithyEventStream, "smithy_eventstream::error"),
         "Header" to RuntimeType("Header", smithyEventStream, "smithy_eventstream::frame"),
         "HeaderValue" to RuntimeType("HeaderValue", smithyEventStream, "smithy_eventstream::frame"),
-        "Inlineables" to RuntimeType.eventStreamInlinables(runtimeConfig),
+        "ExpectFns" to RuntimeType("smithy", smithyEventStream, "smithy_eventstream"),
```

Comment:
generally I favor a module type in lower case for consistency (so it's obvious the template refers to a module and not a struct)